Makefile

```diff
@@ -8,18 +14,21 @@ install: all
 	mkdir -p /usr/share/cockpit/welder
 	cp -r public/* /usr/share/cockpit/welder
 
+# remove evil file name that breaks rpmbuild (https://github.com/patternfly/patternfly/issues/917)
```

Comment:
Comment should be next to the find ... so that when the bug is fixed it's obvious what to revert.

Reply:
Ack, fixed.